app/src/main/java/com/padc/nyi/moneysaver123/activities/AddExpenseActivity.java

```diff
@@ -35,12 +51,73 @@ protected void onCreate(Bundle savedInstanceState) {
         ButterKnife.bind(this, this);
 
         setSupportActionBar(toolbar);
+        ActionBar actionBar = getSupportActionBar();
+        if(actionBar != null){
+            actionBar.setTitle("သံုးလိုက္ေသာ ေငြထည့္ရန္");
+            actionBar.setDisplayHomeAsUpEnabled(true);
+        }
 
-        final String []dummyCategory={"aaa", "bbb"};
+        final String []dummyCategory={"အစားအေသာက္", "အဝတ္အစား", "လမ္းစရိတ္", "ေဖ်ာ္ေျဖေရး", "ကားအသံုးစရိတ္", "အေထြေထြ"};
         ArrayAdapter adapter = new ArrayAdapter(getBaseContext(), R.layout.support_simple_spinner_dropdown_item, dummyCategory);
         adapter.setDropDownViewResource(R.layout.support_simple_spinner_dropdown_item);
         spinnerCategory.setAdapter(adapter);
 
+        getCurrentDate();
     }
 
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        int id = item.getItemId();
+        switch (id){
+            case android.R.id.home:
+                this.finish();
+                break;
+        }
+        return true;
+    }
+
+    @OnClick(R.id.txt_date)
+    public void onClickDate(View view){
+        showThirdPartyDatePicker();
+    }
+    private void getCurrentDate(){
```

Comment:
It's almost always a good option to put such kind of methods in utils class(quite resuable in similar projects) and get the dateString via static method